pkl/private/pkl_codegen_java.bzl

```
@@ -114,6 +115,56 @@ _pkl_java_src_jar = rule(
     ],
 )
 
+def _pkl_java_resource_jar_impl(ctx):
+    srcjar = ctx.file.srcjar
+    resource_jar = ctx.outputs.out
+
+    ctx.actions.run_shell(
+        inputs = [srcjar],
+        outputs = [resource_jar],
+        command = """
+            set -e
+            EXECROOT="$PWD"
+            TMPDIR=$(mktemp -d)
+            trap "rm -rf $TMPDIR" EXIT
+
+            # Extract srcjar
+            unzip -q "$EXECROOT/$1" -d "$TMPDIR" || true
+
+            # Create resource JAR with resources/ prefix stripped
+            cd "$TMPDIR"
+            if [ -d "resources" ]; then
+                cd resources
+                zip -qr "$EXECROOT/$2" . || true
```

What is the logic behind ignoring errors here?